autorest/sender_test.go

```diff
@@ -144,7 +144,7 @@ func TestSend(t *testing.T) {
 func TestAfterDelayWaits(t *testing.T) {
 	client := mocks.NewSender()
 
-	d := 5 * time.Millisecond
```

Comment:
@garimakhulbe I purposely chose Millisecond to keep the tests from getting too long. Were those values not working of you?